CHANGELOG.md

```diff
@@ -57,6 +57,9 @@ Main (unreleased)
 - `loki.source.kafka` component now exposes internal label `__meta_kafka_offset`
   to indicate offset of consumed message. (@hainenber)
 
+- Add a`start_from_eof` attribute in `loki.source.file` to have the option to start tailing a file from the end if a cached position is not found.
```

Comment:
Nit: do we believe eof is a widely understood term for non-developers? If we do, then this is fine, otherwise we might want to pick a different attribute name.

Reply:
@clayton-cornell would you happen to have any naming suggestions here?

Reply:
Some suggestions:

```
- ignore_previous_lines
- ignore_past_content
- skip_past_content
```

Reply:
Another way of doing it: 

```
initial_position = "end"  // or "start" for current behavior 
```

Reply:
The audience we target at this level of configuration has to be fairly knowledgeable. Using `start_from_eof` doesn't raise a big flag for me.  Plus... it is defined in the **Arguments** section of the doc. We have lots of obscure references to very specific tooling throughout Agent.

If we really want to change it, it should be descriptive. Maybe `start_from_end_of_log`?  Or how about `tail_from_end` since it relates directly to the function... the user is tailing the logs`.

Reply:
I'm gonna go with tail_from_end then, unless someone is against it